latextools_utils/is_tex_file.py

```diff
@@ -12,7 +12,6 @@
     strbase = str
 
 def get_tex_extensions():
-    view = sublime.active_window().active_view()
```

Comment:
I removed this, because it raises in error when called inside `plugin_loaded` in ST2 and is not used anyway.